cvat/schema.yml

```diff
@@ -8362,10 +8376,12 @@ components:
       enum:
       - annotation
       - ground_truth
+      - consensus
```

Comment:
Shouldn't this be `consensus_replica`? You've already used this phrase in the task parameter description:

> The number of **consensus replica** jobs for each annotation job

Reply:
Ok, let's rename it to `consensus_replica`.